components/system-broker/pkg/graphql/config.go

```diff
@@ -20,13 +20,16 @@ import "github.com/pkg/errors"
 
 type Config struct {
 	GraphqlEndpoint string `mapstructure:"graphql_endpoint"`
-	EnableLogging   bool   `mapstructure:"enable_logging"`
+	// used when system broker acts like an integration system
+	GraphqlOauthEndpoint string `mapstructure:"graphql_oauth_endpoint"`
```

Comment:
Nice catch, maybe it's a leftover. This OAuth endpoint from the config is never used because the actual OAuth endpoint comes from the secret from which the System Broker read his clientID and clientSecret.

Reply:
Isn't that OAuth secured endpoint of director used when system broker acts as integration system? The one in the secret should be the token endpoint for issuing hydra tokens.

Reply:
Yes, my mistake. @DimitarPetrov is right. So the discussed OAuth endpoint of the Director in this config file is part of the unused integration system communication. We'll remove it as well as other stuff that are related to the integration system communication scenario.

Reply:
We've resolved that.